src/core/workspace.c

```diff
@@ -1065,39 +1063,15 @@ focus_ancestor_or_mru_window (MetaWorkspace *workspace,
           meta_window_focus (ancestor, timestamp);
 
           /* Also raise the window if in click-to-focus */
-          if (meta_prefs_get_focus_mode () == META_FOCUS_MODE_CLICK)
+          if (meta_prefs_get_focus_mode () == G_DESKTOP_FOCUS_MODE_CLICK)
```

Comment:
This line should not be changed. `G_DESKTOP_FOCUS_MODE_CLICK` is not defined for marco:
```
core/workspace.c:1066:47: error: ‘G_DESKTOP_FOCUS_MODE_CLICK’ undeclared (first use in this function); did you mean ‘META_FOCUS_MODE_CLICK’?
 1066 |           if (meta_prefs_get_focus_mode () == G_DESKTOP_FOCUS_MODE_CLICK)
      |                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                               META_FOCUS_MODE_CLICK
```